src/adaptors/spark/index.js

```diff
@@ -6,50 +6,67 @@ const utils = require('../utils');
 const { aTokenAbi } = require('../aave-v3/abi');
 const poolAbi = require('../aave-v3/poolAbi');
 
-const ethV3Pools = async () => {
-  const AaveProtocolDataProviderV3Mainnet =
-    '0xFc21d6d146E6086B8359705C8b28512a983db0cb';
+const sparkChains = ['ethereum', 'gnosis'];
+
+/**
+ * protocol data provider address per chain
+ * @type {{ethereum: string, gnosis: string}}
+ */
+const protocolDataProviderAddress = {
+  ethereum: '0xFc21d6d146E6086B8359705C8b28512a983db0cb',
+  gnosis: '0xA98DaCB3fC964A6A0d2ce3B77294241585EAbA6d',
```

Comment:
not working with the imported abi, guess wrong contract

Reply:
Yeah, the gnosis address was wrong, but still running the test fails against when reading from the eth's address

<img width="1536" alt="image" src="https://github.com/DefiLlama/yield-server/assets/1926216/673ed087-9d34-4366-ab9e-6d6cfeb86be1">

Reply:
its rpc related, is fine on server side!